### PR TITLE
added authors name

### PR DESCRIPTION
**Pull request by LovinKerr**
Please check, thanks.

### CODE REVIEW

**Comment by IvanQin** [comment]
Thanks!

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# understanding_jvm_class_loader
\ No newline at end of file
+# understanding_jvm_class_loade
+
+Authors: Ivan Qin, Nicole Hu
```
Comment (IvanQin) [inline README.md]: Could you add the github link to the authors using markdown?

**Review by IvanQin — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1 +1,3 @@
-# understanding_jvm_class_loader
\ No newline at end of file
+# understanding_jvm_class_loade
```
Comment (IvanQin) [inline README.md]: loade -> load?

**Review by IvanQin — CHANGES_REQUESTED** [review]

**Review by IvanQin — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (LovinKerr) [reply README.md]: Changed.

**Pull request merged**